model/ftn/w3ounfmetamd.ftn

```diff
@@ -0,0 +1,2893 @@
+           CALL EXTCDE(10)
+        ENDIF
+      ENDDO
+    
+  !500 CONTINUE
+      CLOSE(NDMI)
+      WRITE(*, 5000) MCNT, N_GBLMETA
+      RETURN
+!
+ 5000 FORMAT(/' Read in: ',I3,' variable metadata entries' /          &
+              '     and: ',I3,' global meta data entries' /)
+!
+ 5010 FORMAT (/' *** WAVEWATCH III ERROR IN W3OUNFMETA : '/           &
+               '     ERROR OPENING METADATA FILE'/                    &
+               '     FILENAME = ', A /                                &
+               '     IOSTAT =', I5 /)
+!
+ 5011 FORMAT (/' *** WAVEWATCH III ERROR IN W3OUNFMETA : '/           &
+               '     UNKNOWN FIELD ID: ',A /                          &
+               '     FILENAME = ', A /                                &
+               '     LINE =', I5 /)
+!
+ 5012 FORMAT (/' *** WAVEWATCH III ERROR IN W3OUNFMETA : '/           &
+               '     SYNTAX ERROR ' /                                 &
+               '     FILENAME = ', A /                                &
+               '     LINE =', I5 /                                    &
+               '  => ', A /)
+!
+ 5013 FORMAT (/' *** WAVEWATCH III ERROR IN W3OUNFMETA : ' /          &
+               '     IFI value should be in range 1,',I2 /            &
+               '     FILENAME = ', A /                                &
+               '     LINE =', I5 /                                    &
+               '  => ', A /)
+!
+ 5014 FORMAT (/' *** WAVEWATCH III ERROR IN W3OUNFMETA : ' /          &
+               '     IFJ value should be in range 1,',I2 /            &
+               '     FILENAME = ', A /                                &
+               '     LINE =', I5 /                                    &
+               '  => ', A /)
+!
+ 5015 FORMAT (/' *** WAVEWATCH III ERROR IN W3OUNFMETA : ' /          &
+               '     IFC value should be in range 1,3' /              &
+               '     FILENAME = ', A /                                &
+               '     LINE =', I5 /                                    &
+               '  => ', A /)
+!
+      END SUBROUTINE READ_META
+
+!/ ------------------------------------------------------------------- /
+      SUBROUTINE DECODE_HEADER(BUF, ILINE, IFI, IFJ, IFC)
+!/
+!/                  +-----------------------------------+
+!/                  | WAVEWATCH III           NOAA/NCEP |
+!/                  |           C. Bunney               |
+!/                  |                                   |
+!/                  |                        FORTRAN 90 |
+!/                  | Last update :         09-Nov-2020 |
+!/                  +-----------------------------------+
+!/
+!/    09-Nov-2020 : Creation                            ( version 7.XX )
+!/
+!
+!  1. Purpose :
+!
+!     Decode the META header line.
+!
+!  2. Method:
+!
+!     The internal WW3 field can be speicifed either as an [IFI, IFJ]
+!     integer combination, or a field ID tag (such as "HS").
```

Comment:
At the moment, the "META" tag and the "field names" are case sensitive. I did wonder about making them case-insensitive; do you think it would be better? I shouldn't be too difficult to do (assuming an ASCII character set).

Reply:
in w3servmd there is a function STR_TO_UPPER that could call to automatically convert it to upper case.

Reply:
Perfect! I will use that.

Reply:
OK - I've made the META keyword and the "field names IDs" case insensitive in the ounfmeta.inp file: 2612878b